### PR TITLE
Fixing parser mostly

### PR DESCRIPTION
**Pull request by hodak**
I fixed parser, it passes every test except schedule_5 (which was marked as pending) and search - but it didn't pass before either. Will look at it in the future.

Also changed root to /home passing over note about lack of support and added support e-mail link.

### CODE REVIEW

File: config/initializers/mime_types.rb [hunk 1]
```diff
@@ -6,5 +6,4 @@
 Mime::Type.register "application/pdf", :pdf
 Mime::Type.register_alias "application/pdf", :pdfmini
 Mime::Type.register_alias "text/html", :htmlmini
-Mime::Type.register "text/plain", :ics
```
Comment (teamon) [inline config/initializers/mime_types.rb:9]: Why?

File: lib/epure/parser.rb [hunk 2]
```diff
@@ -92,29 +97,32 @@ def parse!
...
+                           entry_time(x) == et
+                       end
+                     end
+                   end.uniq do |e|
+                     entry_time(e)
+                   end.uniq do |e|
+                     [e.week_day, e.building, e.room]
+                   end.each do |e|
+                     entries << e
+                   end
```
Comment (teamon) [inline lib/epure/parser.rb:125]: This parses need some serous refactoring to multiple methods ;)

File: config/initializers/mime_types.rb (hunk 1 above)
Comment (hodak) [inline config/initializers/mime_types.rb:9]: There was a warning about mime type ICS being declared twice.

Comment (teamon) [inline config/initializers/mime_types.rb:9]: Does downloading .ics still work?

File: lib/epure/parser.rb (hunk 2 above)
Comment (hodak) [inline lib/epure/parser.rb:125]: It does, it's on todo-next list but I wanted to get it up, passing tests and running ;)

File: config/initializers/mime_types.rb (hunk 1 above)
Comment (hodak) [inline config/initializers/mime_types.rb:9]: Yes

File: lib/epure/parser.rb (hunk 2 above)
Comment (jagub2) [inline lib/epure/parser.rb:125]: Strange... when I'm using parser from this revision, schedule that I'm getting is empty, anyway parser from previous revision is giving valid schedule.

Comment (hodak) [inline lib/epure/parser.rb:125]: I agree, it's broken. It passes test locally, but it breaks on deploy. I'm still trying to figure out why it could be.

Comment (teamon) [inline lib/epure/parser.rb:125]: Let's see ;)

Comment (teamon) [inline lib/epure/parser.rb:125]: I guess this is fixed now

**Pull request merged**